File: lib_network/src/main/java/com/bryanrady/lib_network/interceptor/CommonResponseInterceptor.java
package com.bryanrady.lib_network.interceptor;

import android.util.Log;

import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.Response;

/**
 * author : bryanrady
 * e-mail : bryanrady@163.com
 * date   : 2021/7/18 16:31
 * desc   :
 */
public class CommonResponseInterceptor implements Interceptor {

    private static final String TAG = CommonResponseInterceptor.class.getSimpleName();

    @Override
    public Response intercept(Chain chain) throws IOException {
        long requestTime = System.currentTimeMillis();
        Response response = chain.proceed(chain.request());
        Log.d(TAG, "requestTime: " + (System.currentTimeMillis() - requestTime));
        return response;
    }
}

File: lib_network/src/main/java/com/bryanrady/lib_network/interceptor/CommonResponseInterceptor.kt
package com.bryanrady.lib_network.interceptor

import android.util.Log
import okhttp3.Interceptor
import okhttp3.Response

/**
 *    author : bryanrady
 *    e-mail : bryanrady@163.com
 *    date   : 2021/8/4 22:56
 *    desc   :
 */
class CommonResponseInterceptor : Interceptor {

    override fun intercept(chain: Interceptor.Chain): Response {
        val requestTime = System.currentTimeMillis()
        val response = chain.proceed(chain.request())
        Log.d("bryanrady", "requestTime: " + (System.currentTimeMillis() - requestTime))
        return response
    }

}